service/service.go

```diff
@@ -136,9 +136,10 @@ func New(config Config) (*Service, error) {
 			Logger:        config.Logger,
 			TenantCluster: tenantCluster,
 
-			CRDLabelSelector:   config.Viper.GetString(config.Flag.Service.CRD.LabelSelector),
-			GuestUpdateEnabled: config.Viper.GetBool(config.Flag.Service.Tenant.Update.Enabled),
-			ProjectName:        project.Name(),
+			CRDLabelSelector:      config.Viper.GetString(config.Flag.Service.CRD.LabelSelector),
+			GuestUpdateEnabled:    config.Viper.GetBool(config.Flag.Service.Tenant.Update.Enabled),
+			PodSecurityPolicyName: config.Viper.GetString(config.Flag.Service.PodSecurityPolicy.Name),
+			ProjectName:           project.Name(),
```

Comment:
Just for the future. With the `pkg/project` package we do not need to inject that anywhere anymore. We can simply use the package where needed. The project package should simply be imported directly where necessary.

Reply:
Thanks for mentioning. I might have missed that during flattening. I'll be sure to resolve in a separate PR.

Reply:
Started here https://github.com/giantswarm/kvm-operator/pull/824